lib/Commands/NinjaBuildCommand.cpp

```diff
@@ -146,7 +146,7 @@ struct BuildExecutionQueue {
   bool useLIFO;
 
 public:
-  BuildExecutionQueue(unsigned numLanes, bool useLIFO)
+  BuildExecutionQueue(unsigned long numLanes, bool useLIFO)
```

Comment:
... actually, that is generally my feeling about _most_ of the changes in this PR, I would rather avoid long (and cast it away, when necessary), or use `uint64_t` in the few cases where we need the precision.